### DISCUSSION TITLE
网盘文件删除后索引内不会清除

### DISCUSSION THREAD

**Discussion by YesterdayNEO** [AlistGo/alist#4623] [Q&A] [answered]
在网盘内删除文件后，索引内仍能搜到该文件，且点进去显示object not found，这种情况该怎么处理？

**Comment by anwen-anyi** [comment, marked as answer]
是的不会清除，因为索引是写进了数据库.它也不可能实时监测你是否 `增删改`了文件
 - 如何处理失效的应该是只能重新重构一份索引了吧？（别的办法我暂时不知道了）
